snapshots/education/2023-08-09/numeracy_gender.xlsx.dvc

```diff
@@ -0,0 +1,26 @@
+meta:
+  name: Historical gender inequality numeracy estimates, 1810 - 1960 (Clio Infra)
+  publication_year: 2013
+  publication_date: '2013-12-26'
+  source_name: Baten, J. via Clio Infra
+  source_published_by: Baten, J. Gender Equality of Numeracy (2013) via Clio Infra
+  url: https://clio-infra.eu/Indicators/GenderEqualityofNumeracy.html
+  source_data_url: https://clio-infra.eu/data/GenderEqualityofNumeracy_Compact.xlsx
+  license_url: https://clio-infra.eu/index.html#about
+  license_name: ''
+  date_accessed: 2023-08-09
+  is_public: true
+  description: |
+    Age heaping refers to the tendency for age data collected in surveys or censuses to be disproportionately reported at certain "attractive" or "round" numbers. This phenomenon often manifests itself in ages being reported at numbers that end in 0 or 5.
+
+    For example, when people are asked their age, they might be more likely to report it as 30, 35, 40, etc., rather than the exact age of 31, 36, or 39.
+
+    The authors of this dataset use this “age heaping” to measure gender inequality in cognitive ability in quantitative reasoning, or “numeracy” and construct a database of age heaping-based estimates of basic numeracy with broad geographic and temporal coverage.
+
+    The higher the measure of gender equality, the lower the share of women rounding up or down their age in comparison to men rounding up or down in a certain country. A positive (negative) gender equality index implies a female (male) numeracy advantage.
```

Comment:
Nice. I'm not sure if it would apply here but just be aware of [this issue](https://github.com/owid/owid-issues/issues/1050) when making charts for gender ratio variables.